response.go

```diff
@@ -0,0 +1,23 @@
+package rocket
+
+type Response struct {
+	headers []Header
+	body    interface{}
+}
+
+type Header struct {
+	Key   string
+	Value string
+}
+
+func NewResponse(body interface{}) *Response {
+	return &Response{
+		headers: make([]Header, 0),
+		body:    body,
+	}
+}
+
+func (res *Response) Headers(headers ...Header) *Response {
```

Comment:
Then you don't have to append at here.

Reply:
resolve at 6431581b27589646600d86deb8783690ff15ddf3